Main.cpp

```diff
@@ -33,8 +38,16 @@ int main(int argc, char** argv){
 
     try {
         PreparedGame game(argv[1]);
+        std::ofstream stream = std::ofstream("log.txt");
+        std::string svgrend = "pretty.svg";
+        std::string svgrend2 = "pretty_observer.svg";
+        game.registerRenderer(new HeroTextRenderer());  
+        game.registerRenderer(new ObserverTextRenderer(stream));
+        game.registerRenderer(new CharacterSVGRenderer(svgrend));
+        game.registerRenderer(new ObserverSVGRenderer(svgrend2));
         game.run();
 
     } catch (const JSON::ParseException& e) {bad_exit(4);}
```

Comment:
ide raknék be még catcheket, mivel így csak a ParserExceptiont kapja el, a többi fajtát nem